frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -216,6 +216,44 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
       },
     };
 
+    const rbacName = `${selectedTargetNamespace}-prometheus`;
+    const role = {
```

Comment:
I'd give this a more specific name so on line 293 it's a little more clear why we're creating the role.

```suggestion
    const prometheusRole: K8sResourceKind = {
```

Reply:
wanted to use it but the `rules` field is not assignable `K8sResourceKind`:

`Type '{ kind: string; apiVersion: string; metadata: { name: string; namespace: any; }; roleRef: { kind: string; name: string; apiGroup: string; }; subjects: { kind: string; name: string; namespace: string; }[]; }' is not assignable to type 'K8sResourceKind'.
  Object literal may only specify known properties, and 'roleRef' does not exist in type 'K8sResourceKind'.`

Reply:
Ah, ideally we'd create `RoleKind` and `RoleBindingKind` with the right properties, but don't worry about it for this PR.